frontend/components/features/objectives/FrequencyLogDialog.tsx

```
@@ -0,0 +1,211 @@
+                <SelectItem value={yesterday}>Ayer ({yesterday})</SelectItem>
+              </SelectContent>
+            </Select>
+          </div>
+
+          {/* Status selector */}
+          <div className="space-y-2">
+            <Label htmlFor="status-selector">Estado</Label>
+            <div
+              id="status-selector"
+              className="grid grid-cols-3 gap-3"
+              role="group"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<fieldset>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)